drivers/modem/simcom-sim7080.h

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2021 metraTec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SIMCOM_SIM7080_H
+#define SIMCOM_SIM7080_H
+
+#include <kernel.h>
```

Comment:
I believe some of these header are not required.

A lot of headers are included when you include `zephyr.h`, i.e. `kernel.h` and probably `errno.h`